plugin/src/main/java/com/denizenscript/denizen/scripts/commands/player/FakeSpawnCommand.java

```diff
@@ -54,79 +56,49 @@ public FakeSpawnCommand() {
     // Use to show a fake creeper in front of the attached player.
     // - fakespawn creeper <player.location.forward[5]>
     //
+    // @Usage
+    // Use to spawn and mount a fake armor stand to the player.
+    // - fakespawn armor_stand <player.location> mount_to:<player>
+    //
     // -->
 
     @Override
     public void addCustomTabCompletions(TabCompletionsBuilder tab) {
         TabCompleteHelper.tabCompleteEntityTypes(tab);
     }
 
-    @Override
-    public void parseArgs(ScriptEntry scriptEntry) throws InvalidArgumentsException {
-        for (Argument arg : scriptEntry) {
-            if (!scriptEntry.hasObject("players")
-                    && arg.matchesPrefix("to", "players")) {
-                scriptEntry.addObject("players", arg.asType(ListTag.class).filter(PlayerTag.class, scriptEntry));
-            }
-            else if (arg.matchesPrefix("d", "duration")
-                    && arg.matchesArgumentType(DurationTag.class)) {
-                scriptEntry.addObject("duration", arg.asType(DurationTag.class));
-            }
-            else if (!scriptEntry.hasObject("entity")
-                    && arg.matchesArgumentType(EntityTag.class)) {
-                scriptEntry.addObject("entity", arg.asType(EntityTag.class));
-            }
-            else if (!scriptEntry.hasObject("location")
-                    && arg.matchesArgumentType(LocationTag.class)) {
-                scriptEntry.addObject("location", arg.asType(LocationTag.class));
-            }
-            else if (!scriptEntry.hasObject("cancel")
-                    && arg.matches("cancel")) {
-                scriptEntry.addObject("cancel", new ElementTag(true));
-            }
-            else {
-                arg.reportUnhandled();
+    public static void autoExecute(ScriptEntry scriptEntry,
+                                   @ArgName("entity") @ArgLinear EntityTag entity,
+                                   @ArgName("location") @ArgLinear @ArgDefaultNull LocationTag location,

```

Comment:
I believe these two would need legacy out-of-order args handling? (note that `cancel` shouldn't need that as it's a boolean arg).